src/client/search/SearchTermsReducer.js

```diff
@@ -0,0 +1,27 @@
+const SearchTermsReducer = (state = [], action) => {
```

Comment:
It's not got to do with a list component though..I've deliberately not named it with List since that's common practice with reducers.